python/conversion.py

```diff
@@ -0,0 +1,418 @@
+from tkinter import *
+from convertor import *
```

Comment:
![image](https://user-images.githubusercontent.com/44049528/94991404-9ce39980-05a0-11eb-87cf-296182cb2ff8.png)
is there any module named `convertor`?

Reply:
Yes convertor is module